ArticleClass.js

```diff
@@ -0,0 +1,32 @@
+export class ARTICLE {
+  constructor(
+    title,
+    content,
+    writer = Writer[Math.floor(Math.random() * 7)] //'Math.floor(Math.random() * 7)' <= mdn 참고
+  ) {
+    this.title = title;
+    this.content = content;
+    this.writer = writer;
+    this.likeCount = "👍x 0";
+    this.createdAt = new Date().toLocaleString("ko-KR", {
+      timeZone: "Asia/Seoul", // ''toLocaleString''mdn 40분동안 찾아보다가 어지러워서 gpt에게 도움 요청
+    });
+  }
+
+  like() {
+    console.log(`${this.title}을(를) 좋아합니다👍`);
+    let num = Number(this.likeCount[3]);
+    num += 1;
+    this.likeCount = this.likeCount.slice(0, 3) + num;
+  }
+}
```

Comment:
현재 `likeCount`를 문자열로 관리하고 있는데, 이렇게 하면 몇 가지 문제가 발생할 수 있습니다.
- 좋아요가 10개 이상이 되면 this.likeCount[3]이 잘못된 위치를 가리킵니다
- 숫자를 비교하거나 정렬할 때 어려움이 생깁니다

그리고 요구사항을 보면 likeCount는 숫자여야 합니다. 
표시할 때만 이모지를 추가하는 방식을 고려해보세요!

```suggestion
    this.likeCount = 0;  // 숫자로 저장
    this.createdAt = new Date().toLocaleString("ko-KR", {
      timeZone: "Asia/Seoul", // ''toLocaleString''mdn 40분동안 찾아보다가 어지러워서 gpt에게 도움 요청
    });
  }

  like() {
  this.likeCount += 1;
  console.log(`${this.title}을(를) 좋아합니다👍 (현재: 👍x ${this.likeCount})`);
}